Assets/Scripts/Controller/Pickup Pup/PPGameController.cs

```diff
@@ -162,6 +164,21 @@ public bool TrySendDogToScout(Dog dog)
 		}
 	}
 
+	public void SelectDog(Dog dog)
+	{
+		this.selectedDog = dog;
+	}
+		
+	public void SendDogToSlot(Dog dog, DogSlot slot)
+	{
+		slot.Init(dog);
+	}
+
+	public void SendSelectedDogToSlot(DogSlot slot)
+	{
+		SendDogToSlot(this.selectedDog, slot);
+	}
+
```

Comment:
Will make this one private, but think it's worth keeping for additional functionality